src/test/java/pages/SongsPage.java

```diff
@@ -0,0 +1,88 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.time.Duration;
+import java.util.List;
+
+public class SongsPage extends BasePage{
+
+    public SongsPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    @FindBy(css = "#songsWrapper > div > div > div.item-container > table")
+        WebElement songTable;
+    @FindBy(css = "#songResultsWrapper > div > div > div.item-container > table")
+        WebElement searchResultsTable;
+    @FindBy(xpath = "//button[@data-test='view-all-songs-btn']")
+        WebElement buttonViewAll;
+    @FindBy(css = "#songResultsWrapper > header > div.song-list-controls > span > button.btn-add-to")
+        WebElement clickAddTo;
+    @FindBy(css = "#songResultsWrapper > header > div.song-list-controls > div > section.new-playlist > form > button")
+        WebElement submitPlaylistButton;
+    @FindBy(css = "#songResultsWrapper > header > div  form.form-new-playlist > input")
+        WebElement playlistInput;
+
+
+    public void addSongToPlaylist(String callPlaylist, String song) throws InterruptedException{
+        playlistCheck(callPlaylist, song);
+        search(song);
+        buttonViewAll.click();
+        returnAnySong(4, searchResultsTable,1).click();
+        wait.until(ExpectedConditions.elementToBeClickable(clickAddTo)).click();
+        clickPlaylist("Lambs");
```

Comment:
By the way, I don't think this is a good solution to verify 1 song to 1 hardcode playlist. It will fail in some cases if:
1. Your playlist does not have a playlist named Lambs before
2. If the playlist name has been added to the song before
=> Suggestion:
1. Add new a playlist
2. Add a song to new playlist created at step 1
3. Delete playlist for clean up data